deploy/scripts/install-combine.sh

```
@@ -271,6 +307,28 @@ if [[ "${STATE}" != "Uninstall-combine" && -z "${COMBINE_VERSION}" ]] ; then
   error "Combine version is not specified."
 fi
 
+SETUP_OPTS=""
+if [ "${STATE}" != "Uninstall-combine" ] ; then
+  # Every helm command runs after the restart that the Pre-reqs step usually
+  # requires, so record a timeout and reuse it until the install finishes.
+  if [ -z "${HELM_TIMEOUT}" ] ; then
+    if [ -f ${TIMEOUT_FILE} ] ; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)